TPU CI failure

- https://github.com/pytorch/ignite/runs/4484818799

```
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/opt/hostedtoolcache/Python/3.7.12/x64/lib/python3.7/site-packages/torch_xla/__init__.py", line 99, in <module>
    import torch
  File "/opt/hostedtoolcache/Python/3.7.12/x64/lib/python3.7/site-packages/torch/__init__.py", line 198, in <module>
    _load_global_deps()
  File "/opt/hostedtoolcache/Python/3.7.12/x64/lib/python3.7/site-packages/torch/__init__.py", line 151, in _load_global_deps
    ctypes.CDLL(lib_path, mode=ctypes.RTLD_GLOBAL)
  File "/opt/hostedtoolcache/Python/3.7.12/x64/lib/python3.7/ctypes/__init__.py", line 364, in __init__
    self._handle = _dlopen(self._name, mode)
OSError: libmkl_intel_lp64.so.1: cannot open shared object file: No such file or directory
Error: Process completed with exit code 1.
```

--- a/.github/workflows/tpu-tests.yml
+++ b/.github/workflows/tpu-tests.yml
@@ -49,6 +49,7 @@ jobs:
           restore-keys: |
             ${{ steps.get-date.outputs.date }}-pytorch-${{ runner.os }}-${{ matrix.xla-version }}-
 
+      # mkl version fixed due to https://github.com/pytorch/ignite/issues/2350
       - name: Install Torch XLA and others
         run: |
 

--- a/.github/workflows/tpu-tests.yml
+++ b/.github/workflows/tpu-tests.yml
@@ -54,7 +54,7 @@ jobs:
 
           ## Install openblas, mkl, gsutil
           sudo apt-get install -y libopenblas-dev libomp5-10 libomp5
-          pip install mkl requests gsutil
+          pip install mkl==2021.4.0 requests gsutil
 
           ## Install torch & xla
           curl https://raw.githubusercontent.com/pytorch/xla/master/contrib/scripts/env-setup.py -o pytorch-xla-env-setup.py

